modules/plugins/wfly-10/src/main/resources/META-INF/rhq-plugin.xml

```diff
@@ -8709,245 +8709,251 @@
+
+                          </service><!-- End of filter-ref service -->
+
+                          <service name="Undertow-Location" displayName="Location"
+                                            discovery="SubsystemDiscovery"
+                                            class="BaseComponent"
+                                            createDeletePolicy="both">
+
+                                <plugin-configuration>
+                                    <c:simple-property name="path" readOnly="true" default="location"/>
+                                </plugin-configuration>
+
+                              <resource-configuration>
+                                  <c:simple-property name="handler" required="true" type="string" readOnly="false" description="Default handler for this location"/>
+                              </resource-configuration>
+
+                                  <service name="Location Filter Ref" displayName="Undertow Filter Ref"
+                                                    discovery="SubsystemDiscovery"
+                                                    class="BaseComponent"
+                                                    createDeletePolicy="both">
+
+                                        <plugin-configuration>
+                                            <c:simple-property name="path" readOnly="true" default="filter-ref"/>
+                                        </plugin-configuration>
+
+                                      <resource-configuration>
+                                          <c:simple-property name="predicate" required="false" type="string" readOnly="false"
+         description="Predicates provide a simple way of making a true/false decision  based on an exchange. Many handlers have a requirement that they be applied conditionally, and predicates provide a general way to specify a condition."/>
+                                          <c:simple-property name="priority:expr" required="false" type="string" readOnly="false" defaultValue="1"
+         description="Defines filter order, it should be set to 1 or more, higher number instructs server to be included earlier in handler chain than others under same context. The default value is 1."/>
+                                      </resource-configuration>
+
+                                  </service><!-- End of filter-ref service -->
+
+                          </service><!-- End of location service -->
+
+                  </service><!-- End of host service -->
+      </service> <!-- End of Server service -->
+      <service name="Servlet Container"
+                  discovery="SubsystemDiscovery"
+                  class="BaseComponent"
+                  createDeletePolicy="both">
 
       <plugin-configuration>
-        <c:simple-property name="path" readOnly="true" default="virtual-server"/>
+          <c:simple-property name="path" readOnly="true" default="servlet-container=default"/>
```

Comment:
This is restricting the path to a single servlet-container. There can be multiple, so the "default" should not be enforced here.